cmd/timoni/apply.go

```diff
@@ -231,10 +231,18 @@ func runApplyCmd(cmd *cobra.Command, args []string) error {
 
 	exists := false
 	sm := runtime.NewStorageManager(rm)
-	if _, err := sm.Get(ctx, applyArgs.name, *kubeconfigArgs.Namespace); err == nil {
+	instance, err := sm.Get(ctx, applyArgs.name, *kubeconfigArgs.Namespace)
+	if err == nil {
 		exists = true
 	}
```

Comment:
Wouldn't this break upgrades as `instancesOwnershipConflicts` doesn't check the bundle name against the instance label?

Reply:
Not sure if I understand the question.

By _instance label_ I assume you refer to the Bundle tracking label of the Instance(?). Instances deployed outside Bundles are, as of now, expected to have no Bundle tracking label. `instancesOwnershipConflicts` only checks for the presence of any Bundle tracking label on the Instance.

Reply:
Sorry I got confused, you're right we should warn people they are are overriding an instance managed by a bundle. Can we tell them to use `--force` in the error message, if they really want to override?

Reply:
Added a new commit, where ownership transfer has been moved to a separate flag. I guess it was a bit unfortunate to combine this with `--force`, as force apply has other significant consequences.